### PR TITLE
Add function for Laravel Mix manifest

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/Vinelab/Cdn/CdnFacade.php [hunk 1]
```diff
@@ -103,6 +103,27 @@ public function elixir($path)
...
+     *
+     * @param $path
+     *
+     * @return mixed
+     *
+     * @throws Eceptions\EmptyPathException, \InvalidArgumentException
+     */
+    public function mix($path)
+    {
+        static manifest = null;
```
Comment (vincent-kinow) [inline src/Vinelab/Cdn/CdnFacade.php]: $ is missing

File: src/Vinelab/Cdn/CdnFacade.php [hunk 2]
```diff
@@ -103,6 +103,27 @@ public function elixir($path)
...
     }
 
+    /**
+     * This function allow Laravel Mix to call the right rev-manifest for assets.
+     *
+     * @param $path
+     *
+     * @return mixed
+     *
+     * @throws Eceptions\EmptyPathException, \InvalidArgumentException
```
Comment (vincent-kinow) [inline src/Vinelab/Cdn/CdnFacade.php]: Eceptions ?

File: src/Vinelab/Cdn/CdnFacade.php [hunk 3]
```diff
@@ -103,6 +103,27 @@ public function elixir($path)
...
+     */
+    public function mix($path)
+    {
+        static manifest = null;
+        if (is_null($manifest)) {
```
Comment (vincent-kinow) [inline src/Vinelab/Cdn/CdnFacade.php]: Why check if `$manifest` is null ? `$manifest` is set to null just before

Comment (ghost) [reply src/Vinelab/Cdn/CdnFacade.php]: It was a c/c from the elixir() method :)

**Pull request merged**